reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -545,22 +636,12 @@ class StaticHtmlReporter : Reporter {
                                     div {
                                         +finding.license
                                         if (permalink != null) {
-                                            val count = finding.locations.size
-                                            if (count > 1) {
-                                                +" (exemplary "
-                                                a {
-                                                    href = permalink
-                                                    +"link"
-                                                }
-                                                +" to the first of $count locations)"
-                                            } else {
-                                                +" ("
-                                                a {
-                                                    href = permalink
-                                                    +"link"
-                                                }
-                                                +" to the location)"
+                                            +" ("
```

Comment:
Why are you reducing the amount of links to 1?

Reply:
This is part of a forEach loop, so I am not reducing the number of links to one.